### PR TITLE
Javadoc ndg

### PR DESCRIPTION
**Pull request by wwweert123**

### CODE REVIEW

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: src/main/java/seedu/financialplanner/utils/Ui.java [hunk 1]
```diff
@@ -77,14 +77,14 @@ public void printWatchListHeader() {
...
-        System.out.print("                    ");
+        System.out.print("                       ");
         System.out.print("Last Updated");
         System.out.print("     ");
         System.out.println();
     }
 
     public void printWatchListAcknowledgement() {
-        showMessage("Data provided by Financial Modeling Prep and Alpha Vantage 😊");
+        showMessage("Data provided by Financial Modeling Prep and Alpha Vantage =)");
```
Comment (ryan1604) [inline src/main/java/seedu/financialplanner/utils/Ui.java:87]: why remove emoji D:

**Review by ryan1604 — APPROVED** [review]
LGTM

File: src/main/java/seedu/financialplanner/utils/Ui.java (hunk 1 above)
Comment (wwweert123) [reply src/main/java/seedu/financialplanner/utils/Ui.java:87]: doesnt work on terminal

**Pull request merged**